### PR TITLE
Ran autolesshint src/

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
See https://github.com/automutate/autolesshint/issues/65. There are some known issues in both running autolesshint and the resultant modifications. This PR contains the direct result of running `autolesshint src/`, without any fixes to issues in the modified code.

### CODE REVIEW

File: src/app/ansyn/ansyn/ansyn.component.less [hunk 1]
```diff
@@ -13,13 +13,15 @@
 			flex-direction: column;
 			overflow: hidden;
 			position: relative;
-			.maps-container {
+
+.maps-container {
```
Comment (AmitMY) [inline src/app/ansyn/ansyn/ansyn.component.less]: If it finds a rule that is adjacent to another one, it creates a new line, but also trims that line so it ignores indentation. (this is consistent throughout all of the code)

File: src/app/packages/login/login/login.component.less [hunk 2]
```diff
@@ -57,18 +58,20 @@
...
+&:not(:first-child) {
+						margin-top: 23px;
 					}
-					input {
+
+input {
 						width: 220px;
 						background: transparent;
-						border: none;
+						border: noneone;
```
Comment (AmitMY) [inline src/app/packages/login/login/login.component.less]: `noneone` is not a real thing, while `none` is
This also repeats itself

File: src/styles/styles.less [hunk 3]
```diff
@@ -197,8 +199,9 @@ input[type=number]::-webkit-outer-spin-button {
 	font-style: normal;
 }
 
-[class^="icon-"], [class*=" icon-"] {
-	/* use !important to prevent issues with browser extensions that change fonts */
+[class^="icon-"], 
+[class*=' icon-"] {
```
Comment (AmitMY) [inline src/styles/styles.less]: For some reason it did replace the first `"` with `'`, but not the second one

File: src/styles/styles.less [hunk 4]
```diff
@@ -197,8 +199,9 @@ input[type=number]::-webkit-outer-spin-button {
...
+	/* use !imp'rtant to prevent issues with browser extensions that change fonts */
```
Comment (AmitMY) [inline src/styles/styles.less]: It saw the word `!important` in a comment and changed it

**Comment by JoshuaKGoldberg** [comment]
Issues on autolesshint would be great thanks!

**Pull request closed without merging**